killbill/payment_method.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2018 Federico Torresan
```

Comment:
We would ask you to assign copyright to `The Billing Project, LLC` instead.

Reply:
Ok, I add The Billing Project, LLC to the copy line.